mlflow/pipelines/steps/evaluate.py

```diff
@@ -305,7 +319,15 @@ def get_icon(validated):
         shap_plot_tab.add_image("SHAP_BAR_PLOT", shap_bar_plot_path, width=800)
         shap_plot_tab.add_image("SHAP_BEESWARM_PLOT", shap_beeswarm_plot_path, width=800)
 
-        # Tab 3: Run summary.
+        # Tab 3: Warning log outputs.
+        warning_output_path = os.path.join(output_directory, "warning_logs.txt")
+        if os.path.exists(warning_output_path):
+            warnings_output_tab = card.add_tab("Warning Outputs", "{{ STEP_WARNINGS }}")
```

Comment:
Can we change `Warning Outputs` to `Warning Logs` to make it explicit it's part of the execution log?